### DISCUSSION TITLE
Replacement for the REGNIE procedure in 2022

### DISCUSSION THREAD

**Discussion by Weathermann** [Weathermann/radolan2map#6] [Announcements]
An important announcement for our work on the REGNIE grids:
The REGNIE procedure will be **discontinued in the course of 2022** and the grids in this format will not be continued!
The replacement for this data is HYRAS grids, which are also published. The procedure is similar (also purely station-interpolated) with less influence of orography.
These data are available in netCDF format, so that there should no longer be any problems with GIS compatibility.

https://opendata.dwd.de/climate_environment/CDC/grids_germany/multi_annual/hyras_de/precipitation/

**Comment by jamaa** [comment]
Really? Is there an official announcement by DWD for this? And does it also apply to the daily and monthly REGNIE data?

**Reply by Weathermann** [reply]
>Since you seem to have such a good connetion to the DWD :-) : I have noticed that sometimes, older daily and monthly REGNIE gets updated on the DWD server and I have always wondered why and whether there is a place where such retroactive changes are documented?

Unfortunately, I don't know anything about it and I'm no longer involved in this area. :man_shrugging: 
1. You can send your questions to hydromet [at] dwd.de
2. You can also ask for subscription to the Hydromet newsletter, I suspect this is where such announcements are made

**Comment by jamaa** [comment]
I checked out the HYRAS datasets and disagree with your opinion that they should be no problem with GIS compatibility:
* QGIS doesn't recognize the coordinate system (EPSG:3034) properly, it appears as "unknown" or "SOURCECRS" in QGIS, even though, if you load the netcdf file as a raster, it does display it in the correct location automatically.  
* If you load the netcdf filre as a raster, of course you get each timeslice as an individual band, which I always find really hard to work with. It also doesn't allow any time-based display in QGIS.
* If you load the netcdf file as a mesh (as which it should be treated), it doesn't recognize the coordinate system at all and, even worse, doesn't recognize the data variables as datasets. It only recognizes Latitude and Longitude as datasets, which of course is useless. I don't think it recognizes the time dimension properly either.

Perhaps I am doing it wrong? Or perhaps this means we need new plugin functionality for being able to properly display HYRAS in QGIS? :-)

Anyway, if it is true that REGNIE will be discontinued it would be good to know officially, at work we rely on it quite heavily and would have to adjust accordingly in time.

**Reply by jamaa** [reply]
The official announcement has been made (CDC email newsletter). I guess I will be developing scripts for working with HYRAS data next year. :-)
